### DISCUSSION TITLE
Ezviz C6W - flusso rtsp

### DISCUSSION THREAD

**Discussion by robpea72** [indomus/forum#669] [Hardware & Firmware]
Buonasera a tutti,
ho acquistato una camera Ezviz C6W e non riesco ad accere al flusso video locale tramite rtsp. Ho provato utilizzando rtsp://admin:verificationCode@192.168.1.NNN/ in varie 'salse' prendendo spunto dalle url di altre exviz (es rtsp://admin:verificationCode@192.168.1.NNN//h265_stream. Vorrei altresì aggiungerla al mio server motioneye ma non ho idea di come farla riconoscere al server (ho già una dlink che uso da tempo senza problemi...). Qualcuno riesce ad aiutarmi? Grazie in anticipo!

**Comment by KC-inDomus** [comment]
rtsp://admin:verificationCode@192.168.1.xxx:554/video

**Comment by gsarcl366** [comment]
A me funziona con rtsp://admin:verificationCode@192.168.1.xxx:554
